osgp/platform/osgp-secret-management/src/test/java/org/opensmartgridplatform/secretmanagement/application/services/encryption/providers/JreEncryptionProviderTest.java

```diff
@@ -52,113 +28,17 @@ public void setUp() {
   }
 
   @Test
-  public void identityTest() throws EncrypterException {
+  public void shouldDecryptEncryptedSecret() throws EncrypterException {
```

Comment:
The JreEncryptionProvider class also has an encrypt function. Should we add a test for that function as well?

Reply:
Encryption with this algorithm is asymmetric, which makes testing the encryption harder. There is no way to compare with a known value in the assert. I solved this by encrypting and decrypting a secret in the same test. The output should match the input after encryption and successive, decryption.